ceilometer/manifests/init.pp

```diff
@@ -256,7 +256,7 @@
     'publisher/metering_secret'      : value => $metering_secret, secret => true;
     'DEFAULT/debug'                  : value => $debug;
     'DEFAULT/verbose'                : value => $verbose;
-    'DEFAULT/notification_topics'    : value => join($notification_topics, ',');
+    'DEFAULT/notification_topics'    : value => $notification_topics;
   }
```

Comment:
I make the variable "undef". So, if I leave it with the join statement, it gives an error that join cannot work on "undef" variable